toolkit/scripts/srpm_pack.mk

```diff
@@ -17,20 +17,15 @@ SRPM_BUILD_CHROOT_DIR = $(BUILD_DIR)/SRPM_packaging
 SRPM_BUILD_LOGS_DIR = $(LOGS_DIR)/pkggen/srpms
 
 toolchain_spec_list = $(toolchain_build_dir)/toolchain_specs.txt
```

Comment:
We will need to handle the toolchain pack list as well. Maybe convert to a call to:
```make
toolchain_spec_list := $(call shell_real_build_only, ./list_toolchain_specs.sh \
                                                                                    $(SCRIPTS_DIR)/toolchain/build_official_toolchain_rpms.sh \
                                                                                    $(toolchain_spec_list) )
```
That means we probably need `$(STATUS_FLAGS_DIR)/build_toolchain_srpms.flag:` to inherit the dependencies from the old `$(toolchain_spec_list): $(toolchain_files)` this would replace.

Reply:
```make
toolchain_spec_list := $(call shell_real_build_only, ./list_toolchain_specs.sh \
$(SCRIPTS_DIR)/toolchain/build_official_toolchain_rpms.sh \
/dev/stdout )
```
Fix output ^